indra/newview/llluamanager.cpp

```diff
@@ -401,3 +401,213 @@ void LLLUAmanager::runScriptOnLogin()
     runScriptFile(filename);
 #endif // ! LL_TEST
 }
+
+
+bool is_absolute_path(std::string_view path)
+{
+#ifdef LL_WINDOWS
+    // Must either begin with "X:/", "X:\", "/", or "\", where X is a drive letter
+    return (path.size() >= 3 && isalpha(path[0]) && path[1] == ':' && (path[2] == '/' || path[2] == '\\')) ||
+           (path.size() >= 1 && (path[0] == '/' || path[0] == '\\'));
+#else
+    // Must begin with '/'
+    return path.size() >= 1 && path[0] == '/';
+#endif
+}
+
+std::string join_paths(const std::string &lhs, const std::string &rhs)
+{
+    std::string result = lhs;
+    if (!result.empty() && result.back() != '/' && result.back() != '\\')
+        result += '/';
+    result += rhs;
+    return result;
+}
+
+std::string read_file(const std::string &name)
+{
+    llifstream in_file;
+    in_file.open(name.c_str());
+
+    if (in_file.is_open())
+    {
+        std::string text {std::istreambuf_iterator<char>(in_file), std::istreambuf_iterator<char>()};
+        return text;
+    }
+
+    return std::string();
+}
+
+LLRequireResolver::LLRequireResolver(lua_State *L, std::string path) : mPathToResolve(std::move(path)), L(L)
+{
+    lua_Debug ar;
+    lua_getinfo(L, 1, "s", &ar);
```

Comment:
[lua_getinfo](https://github.com/luau-lang/luau/blob/80928acb92d1e4b6db16bada6d21b1fb6fa66265/VM/include/lua.h#L398) and [lua_Debug](https://github.com/luau-lang/luau/blob/80928acb92d1e4b6db16bada6d21b1fb6fa66265/VM/include/lua.h#L415) are a little bit different compared to default Lua.

Reply:
Thank you! I still think both of those links should go into a comment about that line. Not only is the `lua_getinfo()` signature otherwise undocumented, but the Lua documentation also uses uppercase "S" instead of lowercase "s".

Reply:
Agree. Both links above lead to the same file, so I think we can add just one link.